src/engine/controls/loopingcontrol.cpp

```diff
@@ -1967,3 +2058,42 @@ void BeatLoopingControl::slotToggle(double value) {
         emit activateBeatLoop(this);
     }
 }
+
+void BeatLoopingControl::slotReverseActivate(double value) {
+    bool isReverseAnchor = m_pCOLoopAnchor->toBool();
+    if (!isReverseAnchor) {
+        // If the reverse mode is not enabled, we do so temporarily
+        m_pCOLoopAnchor->set(1.0);
```

Comment:
This looks a bit hacky, because m_pCOLoopAnchor may have the side effect that the GUI and controller buttons are updated. Can you pass it as bool parameter?

Reply:
Added the enum - does it work for you?

Reply:
That's even better. Thank you.